fractions/monitor/src/main/java/org/wildfly/swarm/monitor/runtime/HttpContexts.java

```diff
@@ -150,14 +151,13 @@ private void proxyRequests(HttpServerExchange exchange) {
 
                     int i = 0;
                     for (InVMResponse resp : responses) {
-                        if (200 == resp.getStatus()) {
-                            sb.append(resp.getPayload());
-                        } else if (503 == resp.getStatus()) {
-                            sb.append(resp.getPayload());
-                            failed = true;
-                        } else {
-                            throw new RuntimeException("Unexpected status code: " + resp.getStatus());
+
+                        sb.append(resp.getPayload());
+
+                        if (!failed) {
+                            failed = resp.getStatus() != 200;
```

Comment:
I was considering that, but than we again we control the response here https://github.com/wildfly-swarm/wildfly-swarm/blob/master/client-apis/health-api/src/main/java/org/wildfly/swarm/health/HealthResponseFilter.java